### DISCUSSION TITLE
How does keras tuner count objective?

### DISCUSSION THREAD

**Discussion by GustawOhler** [keras-team/keras-tuner#581] [Q&A]
When I tuned my model with Bayesian tuner it seemed to me that it takes the best value across all executions. It didn't fit my needs so I changed to Random tuner and now it seems that it takes some sort of average but I don't know if it's from best scores from all executions or all epochs of all executions. So - like in title - how does the keras tuner count objective for comparing trials?

**Comment by haifeng-jin** [comment]
The best score of a trial is the mean of the best epoch of all executions.
